### PR TITLE
Added configurable rate limit notification message

### PR DESCRIPTION
**Pull request by TobiTenno**
Updates for #11, added doc for #10 to Configuration in rate-limit.coffee, added HUBOT_RATE_LIMIT_NOTIFY_MSG

### CODE REVIEW

File: src/rate-limit.coffee [hunk 1]
```diff
@@ -35,6 +37,8 @@ module.exports = (robot) ->
     # to using the regex to uniquely identify the listener (even though
     # it is dirty).
     listenerID = context.listener.options?.id or context.listener.regex
+    #message
+    rateLimitMsg = process.env.HUBOT_RATE_LIMIT_NOTIFY_MSG || "Rate limit hit! Please wait #{minPeriodMs/1000} seconds before trying again."
```
Comment (michaelansel) [inline src/rate-limit.coffee]: This needs to happen after line 52 to get the right value of `minPeriodMs`

File: src/rate-limit.coffee [hunk 2]
```diff
@@ -4,6 +4,8 @@
 # Configuration:
 #   HUBOT_RATE_LIMIT_NOTIFY_PERIOD - how frequently to put rate limiting messages into chat (accounting done by listener)
 #   HUBOT_RATE_LIMIT_CMD_PERIOD - how frequently to execute any single listener (can be overridden by the listener)
+#   HUBOT_RATE_LIMIT_SILENT - (Optional) Setting this environment variable to any truthy value will supress rate limit exceeded feedback messages in chat.
+#   HUBOT_RATE_LIMIT_NOTIFY_MSG - message to be sent when user has exceeded rate limit
```
Comment (michaelansel) [inline src/rate-limit.coffee]: Minor nit: "(Optional)"

**Comment by michaelansel** [comment]
Just one change and this should be good to merge

**Comment by TobiTenno** [comment]
amended, @michaelansel

**Pull request merged**